fsspec/implementations/local.py

```diff
@@ -65,10 +65,6 @@ def ls(self, path, detail=False, **kwargs):
         else:
             return [posixpath.join(path, f) for f in os.listdir(path)]
 
-    def glob(self, path, **kwargs):
-        path = self._strip_protocol(path)
```

Comment:
Removing _strip_protocol from the flow here is likely the cause of all the windows errors, because for LocalFileSystem, it also converts windows-style paths to posix.

Reply:
This is what caused this issue: https://github.com/fsspec/filesystem_spec/issues/1322
How about using `make_posix_path` helper here then?

Reply:
I fixed the windows tests by using `make_path_posix` on the expected outputs when required.